templates/Procfile

```diff
@@ -1,2 +0,0 @@
-web: bundle exec puma -p $PORT -C ./config/puma.rb
-worker: bundle exec rake jobs:work
```

Comment:
I assume the practical difference is that we use Rails' default (https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/puma.rb) instead of our configuration file (https://github.com/thoughtbot/suspenders/blob/77312c9629535e3ccb279bc7316e77ace4ed0fb4/templates/puma.rb). @derekprior @calebthompson @jferris do we need our custom configuration on Rails 5?